edx_sga/templates/staff_graded_assignment/show.html

```diff
@@ -4,41 +4,48 @@
      data-staff="{{ is_course_staff }}">
   <script type="text/template" id="sga-tmpl">
     <% if (display_name) { %>
-       <b><%= display_name %></b>
-    <% } %>
-    <% if (uploaded) { %>
-      <p><b>{% trans "File uploaded" %}</b> 
-        <a href="<%= downloadUrl %>"><%= uploaded.filename %></a></p>
-    <% } else { %>
-      <p>{% trans "No file has been uploaded." %}</p>
-    <% } %>
-    <% if (graded) { %>
       <p>
-        {% blocktrans %}Your score is <%= graded.score %> / <%= max_score %>{% endblocktrans %}<br/>
-        <% if (graded.comment) { %>
-          <b>{% trans "Instructor comment" %}</b> <%= graded.comment %>
-        <% } %>
-        <% if (annotated) { %><br/>
-          <b>{% trans "Annotated file from instructor" %}</b>
-          <a href="<%= annotatedUrl %>"><%= annotated.filename %></a>
-        <% } %>
+        <b><%= display_name %></b>
       </p>
-    <% } else if (uploaded) { %>
-      {% trans "This assignment has not yet been graded." %}
     <% } %>
     <% if (upload_allowed) { %>
-    <p>
-      <% if (uploaded) { %>
-        {% trans "Upload a different file" %}
-      <% } else { %>
-        {% trans "Upload your assignment" %}
-      <% } %>
-      <div class="upload">
-        <label>{% trans "Select a file" %}
-          <input class="fileupload" type="file" name="assignment"/>
+      <p class="upload">
+        <label>
+          <% if (uploaded) { %>
+            {% trans "Upload a different file" %}
+          <% } else { %>
+            {% trans "Upload your assignment" %}
+          <% } %>
+          <input class="fileupload" type="file" name="assignment" />
         </label>
+      </p>
+    <% } %>
+    <% if (uploaded) { %>
+      <div class="sequential-status-message">
+        <p>
+          <b>{% trans "File uploaded" %}</b>
+          <a href="<%= downloadUrl %>"><%= uploaded.filename %></a>
+        </p>
+        <p>
+          <% if (graded) { %>
```

Comment:
- I think there is no sense of finalizing graded assignment. 
- Graded assignment should mark as finalized or you guys want to give ability to re-upload assignment after it is graded?

cc @pdpinch

Reply:
How can an assignment be graded if it hasn't been submitted/finalized yet?

Reply:
@amir-qayyum-khan a graded assignment wouldn't be finalized. it would be the opposite: a student finalizes their submission, and _then_ it is graded by the course staff. 

@pdpinch re: grading and finalizing, i commented on a possible logic change [here](https://github.com/mitodl/edx-sga/pull/191#issuecomment-350315369). let me know what you think